### ISSUE TITLE
Swift 1.2

### ISSUE THREAD

**Issue by Katafalkas** [strekfus/FloatRatingView#6]
http://stackoverflow.com/questions/28771896/overriding-method-with-selector-touchesbeganwithevent-has-incompatible-type

**Comment by strekfus** [comment]
Thanks @Katafalkas ! Any idea how I should keep things compatible for both versions?

**Comment by hynese** [comment]
I'm also experiencing this issue since upgrading to 8.3

**Comment by hynese** [comment]
So I changed:

`NSSet` to `Set<NSObject>`

and `touches.anyObject()` to `touches.first`

and it compiles for Swift 1.2

Untested!!!

```
verride public func touchesBegan(touches: Set<NSObject>, withEvent event: UIEvent) {
        if let touch = touches.first as? UITouch {
            let touchLocation = touch.locationInView(self)
            self.handleTouchAtLocation(touchLocation)
        }
    }
    
    override public func touchesMoved(touches: Set<NSObject>, withEvent event: UIEvent) {
        if let touch = touches.first as? UITouch {
            let touchLocation = touch.locationInView(self)
            self.handleTouchAtLocation(touchLocation)
        }
    }
    
    override public func touchesEnded(touches: Set<NSObject>, withEvent event: UIEvent) {
        // Update delegate
        if let delegate = self.delegate {
            delegate.floatRatingView(self, didUpdate: self.rating)
        }
    }

```

**Comment by strekfus** [comment]
Fixed for Swift 1.2. Thanks for the help @Katafalkas and @hynese !
